packages/connectors/package.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "@wagmi/connectors",
-  "description": "A collection of connectors for wagmi",
+  "name": "@energyweb/connectors",
```

@nichonien what do you think of adding `wagmi` here as well?
```suggestion
  "name": "@energyweb/wagmi-connectors",
```